lib/host/wasi/inode-macos.cpp

```diff
@@ -26,6 +26,17 @@ inline constexpr bool isSpecialFd(int Fd) noexcept {
   }
 }
 
+inline constexpr __wasi_size_t
+calculateAddrinfoLinkedListSize(struct addrinfo *const Addrinfo) {
+  struct addrinfo *TmpPointer = nullptr;
```

Comment:
move declaration to `for` statement
`for (struct addrinfo *TmpPointer = Addrinfo; TmpPointer != nullptr;`